plugins/janus_audiobridge.c

```diff
@@ -2777,6 +2787,28 @@ void janus_audiobridge_incoming_rtp(janus_plugin_session *handle, int video, cha
 		pkt->silence = FALSE;
 		pkt->length = 0;
 
+		//First check if probation period
+		if(participant->probation == MIN_SEQUENTIAL) {
+			participant->probation--;
+			participant->expected_seq = pkt->seq_number + 1;
+			JANUS_LOG(LOG_WARN, "(participant %"SCNu64") Probation started with ssrc = %"SCNu32", seq = %"SCNu16" \n", participant->user_id, ntohl(rtp->ssrc), pkt->seq_number);
```

Comment:
The `pkt` instance contains the SSRC as well, no need to `ntohl(rtp->ssrc)` again.

Reply:
SSRC value (in pkt) is previously set to 0 value.